react-website/src/App.js

```diff
@@ -15,6 +16,7 @@ function App() {
           <Route index element={<Homepage />}/>
           <Route path="confirmation" element={<Confirmation />} />
           <Route path="form" element={<Form />} />
+          <Route path="admin-sign-in" element={<AdminSignIn />} />
```

Comment:
Can you change this route to something like "admin/sign-in"

Reply:
if you change this make sure to change the path in the header.js file for the button and the drop down menu.